types/src/human_readable_compatibility.rs

```diff
@@ -0,0 +1,132 @@
+use crate::{contracts::*, Groups, URef};
+#[cfg(test)]
+use crate::{AccessRights, Group};
+use alloc::vec::Vec;
+#[cfg(test)]
+use alloc::{collections::BTreeSet, vec};
+use core::{convert::TryFrom, fmt};
+#[cfg(feature = "json-schema")]
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Serialize, Deserialize)]
+#[cfg_attr(feature = "json-schema", derive(JsonSchema))]
+pub struct ContractVersionAndHash {
```

Comment:
Please consider adding a comment that this is the "human readable" variant of the `pub type ContractVersions` - something similar to [this](https://github.com/casper-network/casper-node/pull/4648/files#diff-cdc2bd89ded09dec4dd954bbb9f3fe66d92f0abed3511106f57cf10719eebb8fR41).

Reply:
no longer applicable